skt/executable.py

```diff
@@ -164,9 +163,6 @@ def setup_parser():
     parser.add_argument("-v", "--verbose", help="Increase verbosity level",
                         action="count", default=0)
     parser.add_argument("--rc", help="Path to rc file", required=True)
-    parser.add_argument("--waiving", help=("Hide waived tests."),
-                        type=lambda x: (str(x).lower() == 'true'),
-                        default=True)
     # FIXME Storing state in config file can break the whole system in case
```

Comment:
Question: is there any piece of software that could be using this option? If so, maybe it would be a better idea to accept the option, just make it a no-op?

Reply:
https://gitlab.com/cki-project/pipeline-definition/-/commit/63424d0e0caa71c7c0d216d864640118311b4e8b that change was already merged